internal/testdata/podset_v2.json

```diff
@@ -0,0 +1 @@
+[2,0,!E8,0,!g,8,!DZ,[8,!Bk,[0,"app",0,"plugin1",0,"plugin2",0,"plugin3",0,"plugin4"],0,!i,0,!Ed,8,!I/,[11,{!R:"0a9d2b9e-779e-11e7-b422-42010a8001be"},[0,!E8,0,!Kc,0,!FC,0,!g,0,!i,0,!R]]],8,!r,[8,!FB,[11,{!i:"some-name"},[0,!V,8,!G,[11,{!i:"VAR_1"},[0,!i,8,!E1,[8,!HB,[0,!I,0,!i]]],11,{!i:"VAR_2"},[0,!i,8,!E1,[8,!HB,[0,!I,0,!i]]],11,{!i:"VAR_3"},[0,!i,8,!E1,[8,!HB,[0,!I,0,!i]]]],0,!5,0,!I4,0,!i,8,!El,[8,!Ds,[0,"cpu"]],0,!MD,0,!Mp,8,!HI,[11,{!Ec:"/var/run/secrets/kubernetes.io/serviceaccount"},[0,!Ec,0,!i,0,!Dp]]]],0,!EO,0,!Dd,0,!Dm,0,!Ht,0,!Hw,0,!JE,0,!Ia,0,!Kq,0,!Nb,0,!Gf,8,!C6,[11,{!i:"default-token-hu5jz"},[0,!i,8,!B0,[0,!GB,0,!Fd]]]],8,!B6,[8,!FA,[11,{!t:!Ih},[0,!Hg,0,!Ki,0,!B6,0,!t],11,{!t:"Initialized"},[0,!Hg,0,!Ki,0,!B6,0,!t],11,{!t:!Gl},[0,!Hg,0,!Ki,0,!B6,0,!t],11,{!t:"Ready"},[0,!Hg,0,!Ki,0,!B6,0,!t]],0,!J/,0,!Bi,0,!BD,0,!BE,0,!Do,0,!Es]]
```

Comment:
`/var/run/secrets/kubernetes.io/serviceaccount` should go in the string table, that is pretty common I think